### PR TITLE
Add ebi-migration.html

### PR DESCRIPTION
**Pull request by manics**
image

### CODE REVIEW

File: ebi-migration.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        <p>
+            During March 2020 the EMBL-EBI data centre which hosts the IDR is migrating to a new physical location.
+            From 11th March until roughly the end of the month the following IDR services will be unavailable:
+        </p>
+        <ul stylex="text-align:left;">
+            <li>Virtual Analysis Environment (VAE)</li>
+            <li>FTP submissions</li>
+        </ul>
+        <p>
+            Web and API access to the <a href="https://idr.openmicroscopy.org/">IDR</a> will continue to function, thanks to the provision of a temporary data centre.
```
Comment (mtbc) [inline ebi-migration.html]:
```suggestion
            Web and API access to the <a href="https://idr.openmicroscopy.org/">IDR</a> will continue to function, thanks to the provision of a temporary data center.
```
Are we still doing American spelling?

**Comment by mtbc** [comment]
Could we also have a line or two of whitespace between the cell-y header and the start of the text?

File: ebi-migration.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            <h1 class="hero-main-header">The Image Data Resource is currently running with reduced functionality</h1>
+        </div>
+    </div>
+</header>
+<!-- end marketing header -->
+
+<div class="row">
+    <div class="medium-12 columns">
+        <p>
+            During March 2020 the EMBL-EBI data centre which hosts the IDR is migrating to a new physical location.
```
Comment (mtbc) [inline ebi-migration.html]:
```suggestion
            During March 2020 the EMBL-EBI data center which hosts the IDR is migrating to a new physical location.
```
One here too!

**Comment by manics** [comment]
Couldn't easily figure out how to get a space in, merging as is for deployment.

**Pull request merged**